lib/components/PrizeDetailsCards.jsx

```diff
@@ -54,25 +53,28 @@ const TimeDisplay = (props) => {
   )
 }
 
-const PlayersCard = () => {
-  const [poolChainValues] = useAtom(poolChainValuesAtom)
+// const PlayersCard = () => {
+// const { data: poolChainValues } = usePoolChainValues()
 
-  return (
-    <Card className='mx-4'>
-      <CardTitle>Unique Players</CardTitle>
-      <CardPrimaryText>{`$${poolChainValues.sponsorshipSymbol}`}</CardPrimaryText>
-    </Card>
-  )
-}
+//   return (
+//     <Card className='mx-4'>
+//       <CardTitle>Unique Players</CardTitle>
```

Comment:
This seems like it would be good to have, maybe it's somewhere else and we can delete this commented out code?

Reply:
Unique players is super hard to figure out without the subgraph ☹️